main_game.py

```diff
@@ -0,0 +1,174 @@
+import sys
```

Comment:
there is a main_game.py here and in sim/, which one to run? I have been using the sim/ one.

Reply:
given that one uses the town.png I think that's the final one right?

but why is this module still here then?

Reply:
Same as before :( Fixing now